js/application.js

```diff
@@ -0,0 +1,22 @@
+$(document).ready(function(){
+	
+	var template = function(){
+		$.each(files, function(index, value){
+			//Gets mustache file
+			$.get(value, function(data){
+				var compiledTemplate = Handlebars.compile(data);
+				var html = compiledTemplate({});
+				//Places mustache file in correct location
+				$(divs[index]).html(html);
+			});
+		});
+	};
+	
+	var files = ['js/templates/header.handlebars', 'js/templates/footer.handlebars', 'js/templates/nav.handlebars'];
```

Comment:
I would combine these connected arrays as you could get into trouble if you add another file but not another div.  Also, you don't need the function if you do it this way, just call `$.each()` over the array.